src/mixer/state.ts

```diff
@@ -730,10 +730,19 @@ export const openMicrophone = (
   const state = getState().mixer.mic;
   audioEngine.setMicCalibrationGain(state.baseGain);
   audioEngine.setMicVolume(state.volume);
-
+  // Now to patch in the Mic to the Compressor, or Bypass it.
+  audioEngine.setMicProcessingEnabled(state.processing);
   dispatch(mixerState.actions.micOpen(micID));
 };
 
+export const setMicProcessingEnabled = (enabled: boolean): AppThunk => async (
+  dispatch,
+  _
```

Comment:
Just remove the parameter instead of passing `_`.

```suggestion
  dispatch
```